sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md

```diff
@@ -48,3 +48,11 @@ directive:
     $.properties.id["x-nullable"] = true;
     $.properties.text["x-nullable"] = true;
 ```
+
+### Rename StringIndexType enum
```

Comment:
Once https://github.com/Azure/azure-rest-api-specs/pull/13209 is merged, we don't need the transform. 
I added it here just in case the other PR takes time.